modin/pandas/dataframe.py

```diff
@@ -764,47 +764,10 @@ def cov(
         """
         Compute pairwise covariance of columns, excluding NA/null values.
         """
-        if not numeric_only:
-            return self._default_to_pandas(
-                pandas.DataFrame.cov,
-                min_periods=min_periods,
-                ddof=ddof,
-                numeric_only=numeric_only,
-            )
-        numeric_df = self.drop(
-            columns=[
-                i for i in self.dtypes.index if not is_numeric_dtype(self.dtypes[i])
-            ]
+        return self.__constructor__(
+            query_compiler=self._query_compiler.cov(min_periods=min_periods)
```

Comment:
do you know whether this change is desirable for upstream modin? Should OSS modin move all this logic to the base query compiler? Knowing this will help us manage this change as we rebase.

Reply:
Yes, I think this will help us have different logic for pushdown vs OSS modin.